### PR TITLE
packages: improve utils consistency

### PR DESCRIPTION
**Pull request by sergkudinov**

### CODE REVIEW

File: packages/db/src/schema/list.coffee.md [hunk 1]
```diff
@@ -63,4 +63,4 @@ schemas.map( ({name, owner}) => {
       
 ## Dependencies
 
-    utils = require '@nikitajs/engine/lib/utils'
+    utils = require '@nikitajs/engine/src/utils'
```
Comment (wdavidw) [inline packages/db/src/schema/list.coffee.md:66]: certainly not

File: packages/docker/src/load.coffee.md [hunk 2]
```diff
@@ -91,8 +91,8 @@ console.info(`Image was loaded: ${status}`);
       new_images = {}
       status = false
       log message: 'Comparing new images', level: 'INFO', module: 'nikita/lib/docker/load'
-      if string.lines(stdout).length > 1
-        for image in string.lines stdout.toString()
+      if utils.string.lines(stdout).length > 1
+        for image in utils.string.lines stdout.toString()
```
Comment (wdavidw) [inline packages/docker/src/load.coffee.md:95]: yes, nice catch

**Review by wdavidw — CHANGES_REQUESTED** [review]
make sure you checked all utils (including lxd)

File: packages/db/src/schema/list.coffee.md (hunk 1 above)
Comment (sergkudinov) [reply packages/db/src/schema/list.coffee.md:66]: > make sure you checked all utils (including lxd)

I did

Comment (wdavidw) [reply packages/db/src/schema/list.coffee.md:66]: yes, it must now be utils

Comment (sergkudinov) [reply packages/db/src/schema/list.coffee.md:66]: > certainly not

Do you mean I should apply `utils = require './utils'` even if there is no utils in `db`?

Comment (wdavidw) [reply packages/db/src/schema/list.coffee.md:66]: No, I meant to require "lib" and not "src".

**Pull request merged**